java-frontend/src/main/java/org/sonar/java/ast/visitors/SyntaxHighlighterVisitor.java

```diff
@@ -113,13 +114,13 @@ public void visitNode(Tree tree) {
   public void visitToken(SyntaxToken syntaxToken) {
     String text = syntaxToken.text();
     if (keywords.contains(text)) {
-      highlighting.highlight(start(syntaxToken), end(syntaxToken), "k");
+      highlighting.highlight(start(syntaxToken), end(syntaxToken), TypeOfText.KEYWORD);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "highlight"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)